horton/gbasis/test/test_iobas.py

```diff
@@ -139,3 +158,20 @@ def test_gobasis_contraction():
         gbc2.get_segmented_bcs()
     with assert_raises(NotImplementedError):
         gbc.normalize()
+
+
+def test_write_gbs():
+    sto3g = GOBasisFamily('original STO-3G', filename=context.get_fn('basis/sto-3g.gbs'))
+    sto3g.load()
+    new_gobasis = write_gbs('./temp_test_write_gbs.gbs', sto3g)
```

Comment:
Please, use the tmpdir context manager when you write temporary files. See for example https://github.com/theochem/horton/blob/master/horton/io/test/test_xyz.py#L54